crates/frost/src/aggregator.rs

```diff
@@ -304,8 +310,10 @@ impl FrostAggregator {
         message: &[u8],
         tweak: Option<&[u8]>,
     ) -> Result<Signature, AggregatorError> {
-        self.sign_round_1(user_id.clone(), message, tweak).await?;
-        self.sign_round_2(user_id.clone()).await?;
+        let session_id = Uuid::new_v4().to_string();
```

Comment:
```suggestion
        let session_id = global_utils::common_types::get_uuid().to_string();
```

and import `global_utils` in `Cargo.toml`

Reply:
and merge with upstream dev
there issue with
```
#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq, Hash)]
   |          ^^^^^^^^^ the trait `Serialize` is not implemented for `bitcoin::Txid`
```
is fixed